autoparams/src/main/java/org/javaunit/autoparams/CharacterGenerator.java

```diff
@@ -0,0 +1,19 @@
+package org.javaunit.autoparams;
+
+import java.util.Optional;
+
+public class CharacterGenerator implements ObjectGenerator {
+
+    @Override
+    public Optional<Object> generate(ObjectQuery query, ObjectGenerationContext context) {
+        Class<?> type = query.getType();
+        return type.equals(char.class) || type.equals(Character.class)
+            ? factory()
+            : Optional.empty();
+    }
+
+    private Optional<Object> factory() {
+        return Optional.of((char) RANDOM.nextInt(Character.MAX_VALUE + 1));
```

Comment:
```suggestion
        int exclusiveCharUpperBound = Character.MAX_VALUE + 1;
        return Optional.of((char) RANDOM.nextInt(exclusiveCharUpperBound));
```

사소한 의견입니다만, `exclusiveCharacterUpperBound` 라는 이름의 로컬 변수를 사용하면 코드의 의도가 들어나서 좋을 것 같습니다. :)

Reply:
네! 의견 주신것을 반영해보았는데 확실히 의도가 잘 드러나네요. 감사합니다.
혹시, `exclusiveCharUpperBound`와 같은 변수를 상수 필드로 추출하는 것은 어떻게 생각하시나요?

Reply:
@sogoagain 상수 필드로 추출하면 어떤 장점이 있을까요?

Reply:
@jwChung 변하지 않는 값이라 습관적으로 상수 필드로 추출하는 것은 어떨까 생각해보았는데, 현재 상황에서는 이렇다할 장점이 없을 것 같습니다. :)
추출한 로컬 변수에 `final` 키워드를 붙여 변하지 않는 값이란 것을 나타내는 정도가 좋을 것 같단 생각이 드네요.

Reply:
상수 필드로 써도 좋을 것 같습니다. 저도 큰 차이는 못 느끼겠습니다.